modules/System Admin/emailTemplates_manage_duplicate.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/*
+Gibbon, Flexible & Open School System
+Copyright (C) 2010, Ross Parker
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+use Gibbon\Forms\Form;
+use Gibbon\Tables\Action;
+use Gibbon\Services\Format;
+use Gibbon\Comms\EmailTemplate;
+use Gibbon\Domain\User\UserGateway;
```

Comment:
This gateway does not seem to be used in this file...unless I'm missing something.